src/binaries/Cargo.toml

```diff
@@ -55,6 +55,7 @@ common-metrics = { path = "../common/metrics" }
 common-tracing = { path = "../common/tracing" }
 databend-meta = { path = "../meta/service" }
 databend-query = { path = "../query/service" }
+enterprise = { path = "../query/dcl" }
```

Comment:
I think I would avoid using a enterprise crate for the part involving oss.

Maintaining a distinct package for the Enterprise Edition might be more suitable.